src/Comparer/StubValueComparer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Diff\Comparer;
+
+/**
+ * @since 3.0
+ *
+ * @license GPL-2.0+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class StubValueComparer implements ValueComparer {
```

Comment:
I can see that for null implementations, but this is not one. Well, it's very close to being one, but as I tried to say I'm very concerned about the confusing contract of this specific implementation. Not sure if this even qualifies as a "stub". The compare method lies, and not even in a predictable way (like, for example, always false).